Standard pachinko (#76)
MrLongbottom/P10-Article: sections/appendix/cat_auth_pachinko.tex

		Taxonomy \acrlong{pam} & 0.660 \\
		\textbf{Author \acrlong{pam}} & \textbf{0.598} \\
		\textbf{Category \acrlong{pam}} & \textbf{0.585} \\
		\textbf{\acrlong{pam}} & textbf{0.670}
	\end{tabular}
	\label{tab:cat_auth_pachinko_results}
\end{table}

> For comparison we also ran a four level \gls{pam} without any metadata, using 100 and 90 as the sizes of the two middle layers.

Reviewer: ```suggestion
For comparison we also ran a four-level \gls{pam} without any metadata, using 100 and 90 as the sizes of the two middle layers.
```

This ended up providing very good results, slightly better than the Taxonomy \gls{pam}.
This also shows that while Category \gls{pam} and Author \gls{pam} get better results than their \gls{lda} counterparts, it is better to run \gls{pam} without modifications.

> This could be due to our application, but it might also point towards these two particular metadata fields not being particularly useful in this specific application.

Reviewer 2: Hvad menes der specifikt med "application"?

Author: jeg mente at du kunne bare være fordi at vores metoder til at incorporere metadata er skrald

Author: ```suggestion
This could be due our models being unable to make good use of the extra information provided by the metadata, but it might also point towards these two particular metadata fields not being particularly useful in this specific project.
```

Reviewer: ```suggestion
This could be due to our models being unable to make good use of the extra information provided by the metadata.
It might also point towards these two particular metadata fields not being particularly useful in this specific project.
```

> The category metadata are generally very vague and some categories have seemingly no connection between documents.

Reviewer: ```suggestion
The category metadata is generally very vague and some categories have seemingly no connection between documents.
```

> And as we have discussed earlier author metadata might not be as usefull within journalism as authors don't write about the same topics as much as with scientific papers.

Reviewer: ```suggestion
As discussed earlier, the author metadata might not be as useful within journalism as authors don't write about the same topics as with scientific papers.
```

And while the \gls{pam} without metadata does achieve better topic coherence than the Taxonomy \gls{pam}, they are close enough in both topic coherence and manual inspection of the quality of topics that no conclusions can be drawn.

> It could be that Taxonomy \gls{pam} only needed more training time as it is a five level model which increases the training time and complexity significantly compared to the four level model.

Reviewer 2: ```suggestion
It could be that the Taxonomy \gls{pam} only needed more training time as it is a five-level model, which increases the training time and complexity significantly compared to the four-level model.
```
Da den anden kommentar blev outdated

Reviewer 2: Eller måske skulle sætningen bare slettes/ændres, nu hvor plottet viser at den er tæt på at konvergere.

> It could be that Taxonomy \gls{pam} only needed more training time as it is a five level model which increases the training time and complexity significantly compared to the four level model.

Author: ```suggestion
```

> It could be that Taxonomy \gls{pam} only needed more training time as it is a five level model which increases the training time and complexity significantly compared to the four level model.

Reviewer: ```suggestion
It could be that Taxonomy \gls{pam} only needed more training time as it is a five-level model which increases the training time and complexity significantly compared to the four-level model.
```

Reviewer 2: ```suggestion
It could be that the Taxonomy \gls{pam} only needed more training time as it is a five-level model, which increases the training time and complexity significantly compared to the four-level model.
```
